integration/spark/src/main/java/io/openlineage/spark/agent/lifecycle/LogicalPlanSerializer.java

```diff
@@ -36,16 +36,15 @@
 @Slf4j
 class LogicalPlanSerializer {
   private final ObjectMapper mapper;
-  private static final LogicalPlanSerializer instance = new LogicalPlanSerializer();
 
-  public static LogicalPlanSerializer getInstance() {
-    return instance;
-  }
-
-  private LogicalPlanSerializer() {
+  public LogicalPlanSerializer() {
     mapper = new ObjectMapper();
     try {
-      mapper.registerModule(DefaultScalaModule$.MODULE$);
+      mapper.registerModule(
```

Comment:
this code tries to load class dynamically, but it would always fail, cause at runtime we have unprefixed classname
`com.fasterxml.jackson.module.scala.DefaultScalaModule$`
vs expected
`openlineage.com.fasterxml.jackson.module.scala.DefaultScalaModule$`